pkgs/top-level/all-packages.nix

```diff
@@ -34484,7 +34484,12 @@ with pkgs;
 
   rtl-ais = callPackage ../applications/radio/rtl-ais { };
 
-  rtl-sdr = callPackage ../applications/radio/rtl-sdr { };
+  inherit (callPackages ../applications/radio/rtl-sdr{})
```

Comment:
There should be a space between the callPackage path argument and the empty attrset.